cpp/includes/UniffiRustCallStatus.h

```diff
@@ -11,67 +11,3 @@ struct RustCallStatus {
   int8_t code;
   RustBuffer error_buf;
 };
-
-namespace uniffi_jsi {
-using namespace facebook;
-using CallInvoker = uniffi_runtime::UniffiCallInvoker;
-
-template <> struct Bridging<RustCallStatus> {
-  static jsi::Value jsSuccess(jsi::Runtime &rt) {
-    auto statusObject = jsi::Object(rt);
-    statusObject.setProperty(rt, "code", jsi::Value(rt, UNIFFI_CALL_STATUS_OK));
-    return statusObject;
-  }
-  static RustCallStatus rustSuccess(jsi::Runtime &rt) {
-    return {UNIFFI_CALL_STATUS_OK};
-  }
-  static void copyIntoJs(jsi::Runtime &rt,
-                         std::shared_ptr<CallInvoker> callInvoker,
-                         const RustCallStatus status,
-                         const jsi::Value &jsStatus) {
-    auto statusObject = jsStatus.asObject(rt);
-    if (status.error_buf.data != nullptr) {
-      auto rbuf = uniffi_jsi::Bridging<RustBuffer>::toJs(rt, callInvoker,
-                                                         status.error_buf);
-      statusObject.setProperty(rt, "errorBuf", rbuf);
-    }
-    if (status.code != UNIFFI_CALL_STATUS_OK) {
-      auto code =
-          uniffi_jsi::Bridging<uint8_t>::toJs(rt, callInvoker, status.code);
-      statusObject.setProperty(rt, "code", code);
-    }
-  }
-
-  static RustCallStatus fromJs(jsi::Runtime &rt,
-                               std::shared_ptr<CallInvoker> invoker,
-                               const jsi::Value &jsStatus) {
-    RustCallStatus status;
-    auto statusObject = jsStatus.asObject(rt);
-    if (statusObject.hasProperty(rt, "errorBuf")) {
-      auto rbuf = statusObject.getProperty(rt, "errorBuf");
-      status.error_buf =
-          uniffi_jsi::Bridging<RustBuffer>::fromJs(rt, invoker, rbuf);
-    }
-    if (statusObject.hasProperty(rt, "code")) {
-      auto code = statusObject.getProperty(rt, "code");
-      status.code = uniffi_jsi::Bridging<uint8_t>::fromJs(rt, invoker, code);
-    }
-    return status;
-  }
-
-  static void copyFromJs(jsi::Runtime &rt, std::shared_ptr<CallInvoker> invoker,
-                         const jsi::Value &jsStatus, RustCallStatus *status) {
-    auto statusObject = jsStatus.asObject(rt);
-    if (statusObject.hasProperty(rt, "errorBuf")) {
-      auto rbuf = statusObject.getProperty(rt, "errorBuf");
-      status->error_buf =
-          uniffi_jsi::Bridging<RustBuffer>::fromJs(rt, invoker, rbuf);
-    }
-    if (statusObject.hasProperty(rt, "code")) {
-      auto code = statusObject.getProperty(rt, "code");
-      status->code = uniffi_jsi::Bridging<uint8_t>::fromJs(rt, invoker, code);
-    }
-  }
-};
-
-} // namespace uniffi_jsi
```

Comment:
This moves into a `RustCallStatusHelper.cpp`, which is part of the generated file. This is because it calls into `Bridging<RustBuffer>`, which is now part of the generated file.